tests/test.nf.test

```diff
@@ -0,0 +1,29 @@
+nextflow_pipeline {
+
+    name "Test pipeline: NFCORE_CREATETAXDB"
+    script "main.nf"
+    tag "pipeline"
+    tag "nfcore_createtaxdb"
+    tag "test"
+
+    test("test_profile") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(
+                    path("${params.outdir}/diamond/database.dmnd"),
+                    path("${params.outdir}/kaiju/database.fmi"),
+                    ).match() },
+                { assert new File("$outputDir/pipeline_info/software_versions.yml").exists() },
+                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() }
```

Comment:
Is there a specififc nf-test reason to why you use `${params.outdir}` in one and `$outputDir` in the other?

Reply:
I copied from the examples on the migration docs... I may have made a mistake here and copied the modules one instead of a pipeline one... will check and use only one.

Reply:
Ampliseq uses `$outputDir` so going with that one